sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -161,6 +173,117 @@ public EventDirectorySteps(
+          webDriverHelpers.waitForPageLoaded();
+          String district = apiState.getCreatedEvent().getEventLocation().getDistrict().getUuid();
+          webDriverHelpers.selectFromCombobox(
+              EVENT_DISTRICT_COMBOBOX_INPUT, DistrictsValues.getNameByUUID(district));
+        });
+
+    When(
+        "I chose Community option in Event Group Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          String community = apiState.getCreatedEvent().getEventLocation().getCommunity().getUuid();
+          webDriverHelpers.selectFromCombobox(
+              EVENT_COMMUNITY_COMBOBOX_INPUT, CommunityValues.getValueFor(community));
+        });
+
+    When(
+        "I chose Region {string} option in Event Group Directory",
+        (String regionOption) -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.selectFromCombobox(EVENT_REGION_COMBOBOX_INPUT, regionOption);
+        });
+
+    When(
+        "I chose District {string} option in Event Group Directory",
+        (String districtOption) -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.selectFromCombobox(EVENT_DISTRICT_COMBOBOX_INPUT, districtOption);
+        });
+    When(
+        "I chose Community {string} option in Event Group Directory",
+        (String communityOption) -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.selectFromCombobox(EVENT_COMMUNITY_COMBOBOX_INPUT, communityOption);
+        });
+
+    When(
+        "I chose {string} option from Relevnce Status filter in Event Group Directory",
+        (String searchCriteria) -> {
+          String searchText = "";
+          switch (searchCriteria) {
+            case "Active groups":
+              searchText = "Active groups";
+              break;
+            case "Archived groups":
+              searchText = "Archived groups";
+              break;
+            case "All groups":
+              searchText = "All groups";
+              break;
+          }
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.selectFromCombobox(EVENT_STATUS_FILTER_COMBOBOX, searchText);
+        });
+
+    When(
+        "I sort all rows by Group ID in Event Group Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_GROUP_ID_SORT);
+        });
+
+    When(
+        "I sort all rows by Group NAME in Event Group Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_GROUP_NAME_SORT);
+        });
+
+    When(
+        "I click on a Export button in Event Group Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_EXPORT_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_EXPORT_BASIC_BUTTON);
+        });
```

Comment:
why clicking on 2 export buttons?

Reply:
![image](https://user-images.githubusercontent.com/97879405/157424498-41e5cb35-62da-42c1-8520-516d8078a189.png)

Reply:
ok, in this case, let's split the logic into 2 steps as we do 2 actions, so in future, we can reuse them (in case of we want to click on detailed export in other places)